slash-command.py

```diff
@@ -52,44 +52,40 @@ def __call__(self, environ, after_response):
             return iterator
 
 
-with open("config/SPconfig.json") as f:
+with open("config/SPconfig.json") as f: #a metric ton of
```

Comment:
Are all of these values necessary? probably not, there should be a refactor here